### PR TITLE
Clarify Globalnet support with OVN

### PR DESCRIPTION
**Pull request by sridhargaddam**
Related to: https://github.com/submariner-io/submariner-website/issues/304
Signed-Off-by: Sridhar Gaddam <sgaddam@redhat.com>

### CODE REVIEW

File: src/content/operations/known-issues/_index.en.md [hunk 1]
```diff
@@ -21,4 +21,5 @@ Currently, Submariner does not support using [custom `ikeport` and `nattport`](.
 * Globalnet is not compatible with Headless Services. Only ClusterIP Services are supported at this time.
 * Globalnet annotates every Service in a cluster at the moment, whether or not it was exported.
 * Gateway Health Check is not available for Globalnet deployments at this time.
+* Currently, Globalnet is not supported with OVN.
```
Comment (nyechiel) [inline src/content/operations/known-issues/_index.en.md:24]:
```suggestion
* Currently, Globalnet is not supported with the OVNKubernetes network plug-in.
```

Comment (sridhargaddam) [reply src/content/operations/known-issues/_index.en.md]: thanks for reviewing @nyechiel

**Review by nyechiel — APPROVED** [review]

File: src/content/operations/known-issues/_index.en.md [hunk 2]
```diff
@@ -21,4 +21,5 @@ Currently, Submariner does not support using [custom `ikeport` and `nattport`](.
 * Globalnet is not compatible with Headless Services. Only ClusterIP Services are supported at this time.
 * Globalnet annotates every Service in a cluster at the moment, whether or not it was exported.
 * Gateway Health Check is not available for Globalnet deployments at this time.
+* Currently, Globalnet is not supported with the OVNKubernetes network plug-in.
```
Comment (nyechiel) [inline src/content/operations/known-issues/_index.en.md:24]:
```suggestion
* Currently, Globalnet is not supported with the OVN network plug-in.
```

**Review by nyechiel — COMMENTED** [review]
@sridhargaddam thinking about this again, l think we can keep it simple and just call it "OVN".

File: src/content/operations/known-issues/_index.en.md (hunk 2 above)
Comment (sridhargaddam) [reply src/content/operations/known-issues/_index.en.md:24]: Ack.

**Review by tpantelis — APPROVED** [review]

**Pull request merged**